activesupport/lib/active_support.rb

```diff
@@ -95,6 +95,7 @@ def self.eager_load!
   cattr_accessor :test_order # :nodoc:
   cattr_accessor :test_parallelization_threshold, default: 50 # :nodoc:
 
+  @error_reporter = ActiveSupport::ErrorReporter.new
```

Comment:
This is changing a bit how the error reporter works. Now it's just always available on `ActiveSupport.error_reporter` and all other ways to access it are just delegators.